### ISSUE TITLE
Cant create groups from admin panel

### ISSUE THREAD

**Issue by SmithPeder** [webkom/lego#1386]
The web-app only supports the creating of interest groups. So if you want to create a normal group you have to do it via shell in prod

**Comment by ivarnakken** [comment]
I believe we have concluded that we don't want this, no? Simply by fear of people creating a bunch of groups. @SmithPeder

**Comment by SmithPeder** [comment]
Correct! Or one could argue the permissions for creating/deleting groups could be set on just webkom! The the feature could be implemented.
